internal/changestream/stream/package_test.go

```diff
@@ -86,3 +97,14 @@ func (s *baseSuite) expectFileNotifyWatcher() chan bool {
 	s.FileNotifier.EXPECT().Changes().Return(ch, nil).MinTimes(1)
 	return ch
 }
+
+func (s *baseSuite) expectMetrics() {
+	s.metrics.EXPECT().ChangesRequestDurationObserve(gomock.Any()).AnyTimes()
+	s.metrics.EXPECT().ChangesCountObserve(gomock.Any()).AnyTimes()
+	s.metrics.EXPECT().WatermarkInsertsInc().AnyTimes()
+	s.metrics.EXPECT().WatermarkRetriesInc().AnyTimes()
+}
```

Comment:
The metrics in the changestream tests were making it REALLY difficult to debug what was going on. I've blackholed them for now. The tests are unit tests, adding the metrics added cross-cutting concerns and made them more like integration tests. At some point we _may_ want to have special methods for the metrics, but not in this guise.